rfc6962-bis.xml

```diff
@@ -486,45 +486,43 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
+            case precert_sct_v2: SignedCertificateTimestampDataV2;
+            case tree_head_v2: TreeHeadDataV2;
+            case signed_tree_head_v2: SignedTreeHeadDataV2;
+            case consistency_proof_v2: ConsistencyProofDataV2;
+            case inclusion_proof_v2: InclusionProofDataV2;
         } data;
     } TransItem;
           </artwork>
         </figure>
         <t>
-          <spanx style="verb">version</spanx> is the earliest version of this protocol to which the encapsulated data structure conforms. This document is v2. Note that <xref target="RFC6962">v1</xref> did not define <spanx style="verb">TransItem</spanx>, but this document provides guidelines (see <xref target="v1_coexistence"/>) on how v2 implementations can co-exist with v1 implementations. Note also that, since each <spanx style="verb">TransItem</spanx> object is individually versioned, the version should be increased only if changes to it are made that are not backwards-compatible. The addition of encapsulated data structures can be done by adding <spanx style="verb">TransType</spanx> values without increasing the version.
+          <spanx style="verb">versioned_type</spanx> is the type of the encapsulated data structure and the earliest version of this protocol to which it conforms. This document is v2.
+        </t>
+        <t>
+          <spanx style="verb">data</spanx> is the encapsulated data structure. The various structures named with the <spanx style="verb">DataV2</spanx> suffix are defined in later sections of this document.
         </t>
         <t>
-          <spanx style="verb">type</spanx> is the type of the encapsulated data structure. (Note that <spanx style="verb">TransType</spanx> combines the v1 type enumerations <spanx style="verb">LogEntryType</spanx>, <spanx style="verb">SignatureType</spanx> and <spanx style="verb">MerkleLeafType</spanx>). Future revisions of this protocol may add new <spanx style="verb">TransType</spanx> values.
+          Note that <spanx style="verb">VersionedTransType</spanx> combines the <xref target="RFC6962">v1</xref> type enumerations <spanx style="verb">Version</spanx>, <spanx style="verb">LogEntryType</spanx>, <spanx style="verb">SignatureType</spanx> and <spanx style="verb">MerkleLeafType</spanx>. Note also that v1 did not define <spanx style="verb">TransItem</spanx>, but this document provides guidelines (see <xref target="v1_coexistence"/>) on how v2 implementations can co-exist with v1 implementations.
         </t>
         <t>
-          <spanx style="verb">data</spanx> is the encapsulated data structure. The various structures named with the <spanx style="verb">DataV2</spanx> suffix are defined in later sections of this document.
+          Future versions of this protocol may add new <spanx style="verb">VersionedTransType</spanx> values for new data structures, but need only do this for existing data structures that are changed in a manner that is backwards-incompatible.
```

Comment:
"new data structures" in my text is intended to mean (as you put it) "new data structures that do not serve the same purpose as one of the predefined ones".

"need only do this for existing data structures" is meant to be interpreted as "need only add new VersionTransType values for existing data structures".  Is that not clear?